tests/manage/pv_services/test_daemon_kill_during_pvc_pod_deletion_and_io.py

```diff
@@ -376,38 +478,36 @@ def test_daemon_kill_during_pvc_pod_deletion_and_io(
         self.run_io_on_pods(io_pods)
         log.info("Started IO on IO pods")
 
-        # Verify pvc deletion has started
-        pvc_deleting = executor.submit(
-            wait_for_resource_count_change,
-            func_to_use=get_all_pvcs,
-            previous_num=initial_num_of_pvc,
-            namespace=namespace,
-            change_type="decrease",
-            min_difference=1,
-            timeout=30,
-            interval=0.01,
-        )
-
-        # Verify pod deletion has started
-        pod_deleting = executor.submit(
-            wait_for_resource_count_change,
-            func_to_use=get_all_pods,
-            previous_num=initial_num_of_pods,
-            namespace=namespace,
-            change_type="decrease",
-            min_difference=1,
-            timeout=30,
-            interval=0.01,
-        )
-
-        assert pvc_deleting.result(), "Wait timeout: PVCs are not being deleted."
-        log.info("PVCs deletion has started.")
-
-        assert pod_deleting.result(), "Wait timeout: Pods are not being deleted."
-        log.info("Pods deletion has started.")
+        # Wait for 1 second before killing daemons. This is to wait for the create/delete operations to start
+        sleep(1)
+
+        # Kill daemons
+        node_and_kill_proc = {}
+        log.info(f"Killing daemons of {daemons_to_kill}")
+        for node_name, pids in nodes_and_pids.items():
+            # Command to kill the daemon
+            kill_cmd = f"oc debug node/{node_name} -- chroot /host kill -9 {pids}"
```

Comment:
Why `kill_daemon()` method is not used here?
Using `kill_daemon()` from disruption_helpers will allow us to handle the killing process for internal/external mode properly (once #3573 is merged).

Reply:
Due to the issue creating more than one debug pod simultaneously. More than one thread will try to create debug pod for the same node. So here the pid of daemons running in the same pod is combined.

Reply:
ack, so we will need to handle external mode case separately. It can be done in separate pull request.